change/@fluentui-react-charting-066e1b92-842b-43c9-8fe9-9c28c6de2315.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31562&commitId=882f9e0918211f3846b385202cbb7f9748af4746&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Basic.hover.chromium.png|67| Changed|
|react-charting-AreaChart.Basic - Dark Mode.hover.chromium.png|407| Changed|

</details>


  <details><summary><b>react-charting-HorizontalBarChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HorizontalBarChart.Basic - Dark Mode.hover.chromium.png|319| Changed|
|react-charting-HorizontalBarChart.Basic.hover.chromium.png|52| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>